src/installer/pkg/sfx/Microsoft.NETCore.App/PackageOverrides.txt

```diff
@@ -132,31 +154,37 @@ System.Runtime.Numerics|4.3.0
 System.Runtime.Serialization.Formatters|4.3.0
 System.Runtime.Serialization.Json|4.3.0
 System.Runtime.Serialization.Primitives|4.3.0
-System.Security.AccessControl|4.4.0
+System.Runtime.Serialization.Xml|4.3.0
+System.Security.AccessControl|6.0.1
 System.Security.Claims|4.3.0
-System.Security.Cryptography.Algorithms|4.3.0
-System.Security.Cryptography.Cng|4.4.0
+System.Security.Cryptography.Algorithms|4.3.1
+System.Security.Cryptography.Cng|5.0.0
 System.Security.Cryptography.Csp|4.3.0
 System.Security.Cryptography.Encoding|4.3.0
-System.Security.Cryptography.OpenSsl|4.4.0
+System.Security.Cryptography.OpenSsl|5.0.0
 System.Security.Cryptography.Primitives|4.3.0
-System.Security.Cryptography.X509Certificates|4.3.0
-System.Security.Cryptography.Xml|4.4.0
```

Comment:
From what I can see, this line was wrong as System.SEcurity.Cryptography.Xml was never inbox.

Reply:
Note that it is "inbox" in `Microsoft.AspNetCore.App`

Reply:
It's in ASP.NET Core but not .NETCore.  Surprising that no-one ever hit this, maybe because the package version was old?  Or folks listen to us and don't actually use it.  I confirmed your analysis in 2.0 - 9.0.  Thanks for catching it and removing it.

Reply:
@eerhardt I see that an entry for it is missing in https://github.com/dotnet/aspnetcore/blob/main/eng/PackageOverrides.txt.

Reply:
It's there: https://github.com/dotnet/aspnetcore/blob/96f53b4cd0d39c378adf18d908d493ac3a7f522e/eng/PackageOverrides.txt#L38

Not sorted 😕

Reply:
Looks like ASP.NET's should be updated to no longer include System.IO.Pipelines, since that is now in NETCore.App.

Reply:
@eerhardt will you take care of updating the PackageOverrides list in aspnetcore or do you want me to file an issue?

Reply:
I've opened https://github.com/dotnet/aspnetcore/issues/56724 to track updating aspnetcore's file.